ls1intum/Artemis
# Enhancement/Trigger the template participation after a push into the test repo

### Checklist

### Motivation and Context



Trigger the template when a solution build completes with submission type TEST.
Otherwise the template result might be outdated e.g. doesn't show added test cases.

### Description


Trigger the template build when a solution build result with type TEST returns.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration > programming exercises > edit in editor
3. Push into the test repository (submit)
4. The solution build should now be running, the template build should NOT be running
5. When the solution build completes, the template build should now be running.
